inc/3rd-party/plugins/autoptimize.php

```diff
@@ -11,6 +11,25 @@
 endif;
 
 if ( class_exists( 'autoptimizeConfig' ) ) :
+	/**
+	 * Deactivate WP Rocket lazyload if Autoptimize lazyload is enabled
+	 *
+	 * @since 3.3.4
+	 * @author Remy Perona
+	 *
+	 * @param string $old_value Previous autoptimize option value.
+	 * @param string $value New autoptimize option value.
+	 * @return void
+	 */
+	function rocket_maybe_deactivate_lazyload( $old_value, $value ) {
+		if ( $value['autoptimize_imgopt_checkbox_field_3'] !== $old_value['autoptimize_imgopt_checkbox_field_3'] && 1 === (int) $value['autoptimize_imgopt_checkbox_field_3'] ) {
```

Comment:
This would prevent notices if someday the option name changes:

```suggestion
		if ( empty( $old_value['autoptimize_imgopt_checkbox_field_3'] ) && ! empty( $value['autoptimize_imgopt_checkbox_field_3'] ) ) {
```